automatic/ssis-vs2019/legal/VERIFICATION.TXT

```diff
@@ -0,0 +1,16 @@
+VERIFICATION
```

Comment:
VERIFICATION file (and whole legal folder) is not required if extension is not embedded and shout be removed